released/packages/coq-geocoq-axioms/coq-geocoq-axioms.2.4.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+synopsis: "A formalization of foundations of geometry in Coq"
+description: "This subpackage contains the axioms."
+maintainer: "Julien Narboux <julien@narboux.fr>"
+authors: ["Michael Beeson <profbeeson@gmail.com>"
+          "Gabriel Braun <gabriel.braun@unistra.fr>"
+          "Pierre Boutry <pierre.boutry@inria.fr>"
+          "Charly Gries <charly.gries@etu.unistra.fr>"
+          "Julien Narboux <narboux@unistra.fr>"
+          "Pascal Schreck <schreck@unistra.fr>"]
+license: "LGPL 3"
+homepage: "http://geocoq.github.io/GeoCoq/"
+bug-reports: "https://github.com/GeoCoq/GeoCoq/issues"
+dev-repo: "git+https://github.com/GeoCoq/GeoCoq.git"
+depends: [ "coq-geocoq-coinc" { = "2.4.0" } ]
```

Comment:
As seen in MetaCoq https://github.com/coq/opam-coq-archive/blob/master/released/packages/coq-metacoq-checker/coq-metacoq-checker.1.0~alpha+8.9/opam#L27 there is the special version `version` to refer to the current version for packages released together.

Reply:
I had a chat with Pierre today, he will adopt the smarter format the next release